router/sources/router.move

```diff
@@ -190,6 +190,8 @@ module router::router {
                 domain_name,
                 registration_duration_secs,
             );
+            // Clear the name in v1
+            domains::force_clear_registration(&get_router_signer(), option::none(), domain_name)
```

Comment:
Are we also handling clearing the v1 primary name somewhere? I'd suggest handling it inside of `force_clear_registration`.